mpls-over-udp-ip/draft-ietf-detnet-mpls-over-udp-ip.xml

```diff
@@ -250,8 +251,7 @@
   <t>
     S-Labels, d-CW and zero or more F-Labels are used as defined in <xref
     target="I-D.ietf-detnet-mpls"/> and are not modified by this document.
-    In case of aggregates the A-Label is treated as an S-Label and it
-    too is not modified.
+    In case of aggregates the A-Label (if present) is not modified as well.

```

Comment:
how about drop sentence and change to " "S-Labels, A-Label (when present), d-CW..."